### ISSUE TITLE
Attar: Error after activation on self-hosted site 

### ISSUE THREAD

**Issue by bobmatyas** [Automattic/themes#6009]
## Steps to replicate

1. Download Blockbase from WPCOM: https://wordpress.com/theme/blockbase
2. Download Attar from WPCOM: https://wordpress.com/theme/attar
3. Install plugins
4. Activate Attar

## Result

Error on activation: `// updater for WordPress.com themes if ( is_admin() ) include dirname( __FILE__ ) . '/inc/updater.php'; `

Shows in frontend and backend:

<img width="1000" alt="Screenshot on 2022-05-17 at 12-08-30" src="https://user-images.githubusercontent.com/45246438/168858821-d019470d-720b-4fc1-b84d-aaea26279b46.png">

![Annotation on 2022-05-17 at 12-13-21](https://user-images.githubusercontent.com/45246438/168859021-c19a81c6-51a8-4e9e-9df3-d2f2da6fce15.png)

On a different test site I got more errors in WP Admin:

<img width="1799" alt="Screenshot on 2022-05-17 at 11-45-29" src="https://user-images.githubusercontent.com/45246438/168859232-fa90361b-04b7-4606-9efb-84ef5acb6a5d.png">

## Expected

Should be able to activate theme without an issue

* User report: 5217599-zen

**Comment by anomiex** [comment]
It's not exactly an error, it's just output.

What's going on is that the theme downloader is producing a `functions.php` without any opening `<?php`, so the content of the file gets served to the browser as text.

This looks like the same thing as #5399; possibly the theme zip file hasn't been updated since that was fixed?

**Comment by mikachan** [comment]
Yeah, you're right @anomiex, the theme zip hasn't had an update since the problem with the missing opening `<?php` tag was fixed. I've just regenerated the zip file, so this should be working now (the working version of the zip should be 0.0.4).
